tests/.env.test

```
@@ -1,15 +1,15 @@
 NEXT_PUBLIC_ENVIRONMENT=development
-NEXT_PUBLIC_SITE_URL=https://develop.jumper.exchange
+NEXT_PUBLIC_SITE_URL=https://develop.jumper.xyz
 NEXT_PUBLIC_GOOGLE_ANALYTICS_TRACKING_ID=G-T6ZDXLLXB5
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_GOOGLE_ANALYTICS_TRACKING_ID key should go before the NEXT_PUBLIC_SITE_URL key

(UnorderedKey)